### PR TITLE
Add an instructions() method

### PR DESCRIPTION
**Pull request by twny**
This is a static function on each 'card' implementation that returns a String with user-facing instructions.

### CODE REVIEW

File: src/models/flashcard.rs [hunk 1]
```diff
@@ -19,9 +19,15 @@ impl FlashCard {
         }
     }
 
+
     pub fn flip_card(&mut self) {
         self.flipped = !self.flipped;
     }
+
+    pub fn instructions() -> String {
+        return String::from(r#"SPACE: Show cards back"#)
```
Comment (BrookJeynes) [inline src/models/flashcard.rs]: Using a &str here is fine

```suggestion
        return String::from("SPACE: Show cards back")
```

File: src/models/fill_in_the_blanks.rs [hunk 2]
```diff
@@ -45,6 +45,11 @@ impl FillInTheBlanks {
     fn next(&mut self) {
         self.blank_index = self.blank_index % self.answers.len();
     }
+
+    pub fn instructions() -> String {
+        // TODO add instructions
+        return String::from("Press Q to quit")
```
Comment (BrookJeynes) [inline src/models/fill_in_the_blanks.rs]: We should keep instructions consistent with the rest

```suggestion
        return String::from("q: Quit application")
```

**Review by BasementTrix — APPROVED** [review]
Probably better than my method.

**Comment by BrookJeynes** [comment]
Fantastic for a first PR!

As per our call, we will be moving some of the other issues mentioned on call to another PR.

**Review by BrookJeynes — APPROVED** [review]
lgtm! congrats :)

**Comment by twny** [comment]
This PR for me is 

* First Rust PR... EVER
* First PR ever on @twny 
* First MERGE ever on @twny

lfg

**Pull request merged**